apps/oc_chef_authz/src/oc_chef_authz_org_creator.erl

```diff
@@ -92,9 +92,9 @@
 
            %% users
            {add_acl,
-            [mk_tl(container, [cookbooks, data, nodes, roles, environments])],
+            [mk_tl(container, [cookbooks, data, nodes, roles, environments, policies])],
             [create, read, update, delete], [{group, users}]},
-
+           {add_acl, [{container, policies}], [read], [{group, clients}]},
```

Comment:
I vote for eventually. We're trying to get to a short as possible release cadence, which means that the APIs and probably database schemas are going to be in flux for a while, which means we might end up having things like `NOT NULL` fields in the eventual form of the tables which don't exist yet, which implies we might have to delete content to upgrade. On the Chef Client and ChefDK side of things, you have to opt-in with multiple feature flags. For this specific API, we're going to change it to look something like this work-in-progress RFC: https://gist.github.com/danielsdeleo/edcd6aad580c47a0e0e6 So tl;dr, requiring a new org is okay for the moment.

Hit me up in zoom or hipchat if you want to talk about that a little more.